src/server/plugins/register-form/index.js

```diff
@@ -17,18 +17,22 @@ exports.register = function(server, options, next) {
   const stateConfig = cookies.encryptedCookies(!server.settings.app.debug);
   stateConfig.options.path = server.realm.modifiers.route.prefix || '/';
 
+  const routeConfig = _.merge({
+    plugins: { crumb: true }
+  }, cookiesNoCacheConfig);
+
   server.state(stateConfig.name, stateConfig.options);
 
   steps.forEach(([key, options]) => {
     server.route({
-      config: _.merge({}, cookiesNoCacheConfig, {id: `register-form:${key}`}),
+      config: _.merge({}, routeConfig, {id: `register-form:${key}`}),
```

Comment:
Presumably to avoid mutating `routeConfig`. It might also be better to use `assign` instead to avoid merging inherited properties.